pkg/advisory/update_test.go

```diff
@@ -135,7 +178,19 @@ func TestUpdate(t *testing.T) {
 			}
 
 			if err == nil {
-				if diff := cmp.Diff(tt.expectedDoc, advisoryDocs.Select().WhereName(tt.req.Package).Configurations()[0]); diff != "" {
+				diff := cmp.Diff(tt.expectedDoc, advisoryDocs.Select().WhereName(tt.req.Package).Configurations()[0], cmp.FilterPath(func(p cmp.Path) bool {
+					// Check if the path is accessing the ID field within the Advisories slice.
+					if len(p) < 2 {
+						return false
+					}
+					if p[len(p)-1].String() == ".ID" {
+						if _, ok := p[len(p)-2].(cmp.SliceIndex); ok {
+							return true
+						}
```

Comment:
Did you say we can remove this?

Reply:
removed in the other test and forgot this one, done